catalog.html

```diff
@@ -129,47 +132,47 @@ <h2>Каталог</h2>
             
                 <ul>
                     <li>
-                        <img src="https://via.placeholder.com/400x700" width="400" height="700" alt="">
+                        <img src="img/img-sedona1.png" width="400" height="700" alt="">
```

Comment:
```suggestion
                        <img src="img/img-sedona1.jpg" width="400" height="700" alt="">
```
Здесь и в остальных карточках тоже - картинки нужно извлечь в формате JPG. Это позволит сэкономить на размере файлов. В [Б16](https://up.htmlacademy.ru/htmlcss/31/criteries#b16) даже есть на это указание - JPEG для фотографий.